internal/component/database_observability/mysql/collector/loki_entry.go

```diff
@@ -19,7 +21,7 @@ func buildLokiEntry(op string, instanceKey string, line string) loki.Entry {
 		},
 		Entry: logproto.Entry{
 			Timestamp: time.Unix(0, time.Now().UnixNano()),
-			Line:      line,
+			Line:      fmt.Sprintf(`level="%s" %s`, model.LabelValue(level), line),
```

Comment:
Is the cast to `model.LabelValue` really needed?

Reply:
thanks